dev-qt/qt3d/qt3d-5.15.2-r11.ebuild

```diff
@@ -28,6 +28,10 @@ DEPEND="${RDEPEND}
 	vulkan? ( dev-util/vulkan-headers )
 "
 
+PATCHES=(
+	"${FILESDIR}/${P}-riscv-atomic.patch" #bug 803470
```

Comment:
It looks like it would be a "long time" downstream patch (which is thankfully quite easy to maintain).
In that case, I think it would be better to use the `${PN}-5.15.2` instead of `${P}`, as it would make bumping easier in future.

Reply:
oops, @arthurzam sorry for highlighting you.. I probably should at pesa
basically, I think it's a quite trivial patch, should be easy to maintain

anyway, thanks for the review/comments

Reply:
agree with @arthurzam's comment

Reply:
Now use ${PN}-5.15.2